machineSetup/setupPerfMachine.ps1

```diff
@@ -27,10 +27,10 @@ EnableSymlinks=Disabled
 "@
 #endregion
 $gitInstallLog > gitInstallInfo.log
-wget https://github.com/git-for-windows/git/releases/download/v2.12.0.windows.1/Git-2.12.0-64-bit.exe -OutFile git-2.12.exe
+wget https://github.com/git-for-windows/git/releases/download/v2.14.1.windows.1/Git-2.14.1-64-bit.exe -OutFile git-2.14.1.exe
```

Comment:
Could you please replace `echo` with `Write-Host`? `echo` tokenizes the string on spaces and prints them one per line.